src/sentry/api/serializers/models/group.py

```diff
@@ -550,15 +559,19 @@ def __init__(
         self,
         environment_func=None,
         stats_period=None,
+        stats_period_start=None,
+        stats_period_end=None,
         matching_event_id=None,
         matching_event_environment=None,
     ):
         super(StreamGroupSerializer, self).__init__(environment_func)
 
         if stats_period is not None:
-            assert stats_period in self.STATS_PERIOD_CHOICES
+            assert stats_period in self.STATS_PERIOD_CHOICES or stats_period == "auto"
```

Comment:
Should `auto` be one of the valid choices that users can use?

Reply:
I think no `stats_period` already implies that and I wouldn't want to add this to the validator

Reply:
Ok.